project/plugins.sbt

```diff
@@ -8,5 +8,5 @@ addSbtPlugin("com.typesafe.sbt"   % "sbt-site"             % "0.8.1")
 addSbtPlugin("net.virtual-void"   % "sbt-dependency-graph" % "0.7.4")
 addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"         % "0.5.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh"              % "0.2.5")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"          % "0.6.10")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"          % "0.6.12")
```

Comment:
latest scala-js version is 0.6.13

https://groups.google.com/d/msg/scala-internals/LGRU1gEwRgQ/7CAnITvBAQAJ